src/preferences.md

```diff
@@ -7,13 +7,26 @@ the Anki menu on a Mac.
 
 ## Basic
 
+**Language**  
+Change your display language. You can help to iomprove translations [here](https://translating.ankiweb.net/)
```

Comment:
```suggestion
Change your display language. You can help to improve translations [here](https://translating.ankiweb.net/)
```

Reply:
Sorry for the mistakes (and thanks for the corrections) :)